### DISCUSSION TITLE
Yet another llm model format to support? AWQ

### DISCUSSION THREAD

**Discussion by dreemur99** [LostRuins/koboldcpp#448] [Ideas]
Im sure you already seen it already but theres a another new model format. AWQ. Claims to be "blazing-fast" with much lower vram requirements. Looks like an almost 45% reduction in reqs.

**Comment by LostRuins** [comment]
Probably won't be adding this on my own, unless llama.cpp also adds it, then koboldcpp will inherit their support.

**Reply by dreemur99** [reply]
makes sense.
